payments/tests/test_views/test_payment_list_create.py

```
@@ -0,0 +1,495 @@
+        assert response.status_code == status.HTTP_403_FORBIDDEN
+
+    def test_list_payments_authenticated_success(self):
+        """Test successful retrieval of payments for authenticated user."""
+        self.client.force_authenticate(user=self.user)
+        PaymentFactory.create_batch(3, user=self.user)
+
+        response = self.client.get(self.url)
+
+        assert response.status_code == status.HTTP_200_OK
+        assert "results" in response.data
+        assert len(response.data["results"]) == 3  # noqa: PLR2004
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)